src/pdf.py

```diff
@@ -67,6 +67,15 @@ def transform_book_content(path_file):
 
         soup = BeautifulSoup(html, "html.parser")
 
+        for node in soup.select('img[src^="images/https://"]'):
+            print('Fix image link "%s"' % node["src"])
+            node['src'] = node["src"][7:]
+            node['data-dark-src'] = node["data-dark-src"][7:]
+
+        for page in soup.select('[id="test-page.md"], [id="kotlin-brand-assets.md-kotlin-night-brand-assets"]'):
```

Comment:
"Kotlin Night brand assets" fully consists of svg images with png inside.
This is not supported by current tool.